<ui version="4.0" >
 <class>MeshCutDialogClass</class>
 <widget class="QWidget" name="MeshCutDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mesh Segmentation</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>12</y>
     <width>114</width>
     <height>242</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="foreRadioButton" >
        <property name="text" >
         <string>Select Foreground</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="backRadioButton" >
        <property name="text" >
         <string>Select Background</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="foregroundLabel" >
          <property name="text" >
           <string>Foreground Color
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPicker" native="1" name="foreColorPicker" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="backgroundLabel" >
          <property name="text" >
           <string>Background Color
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPicker" native="1" name="backColorPicker" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="meshSegmentButton" >
      <property name="text" >
       <string>Start Segmentation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gaussianButton" >
      <property name="text" >
       <string>Colorize Gaussian</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="meanButton" >
      <property name="text" >
       <string>Colorize Mean</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton" >
      <property name="text" >
       <string>Reset Selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="meshlab.qrc" />
 </resources>
 <connections/>
</ui>
